src/app-config.js

```diff
@@ -3,4 +3,4 @@ const CORS_PROXY = "https://cors-anywhere.herokuapp.com/"
 export const REMOTE_BANK_API = CORS_PROXY + "https://serene-escarpment-42577.herokuapp.com/api"
 export const LOCAL_BLOOD_BANK_API = "http://localhost:5001/api"
 
-export const BLOOD_BANK_API = REMOTE_BANK_API
+export const BLOOD_BANK_API = LOCAL_BLOOD_BANK_API
```

Comment:
set to remote?

Reply:
If set to. remote, it won't work because the MySql database is not configured.

It is not trivial to deploy a database on the web